Client/Frontend/Fakespot/Views/FakespotAdView.swift

```diff
@@ -0,0 +1,315 @@
+        }
+        setupLayout()
+        setNeedsLayout()
+        layoutIfNeeded()
+    }
+
+    // MARK: Subviews Setup
+    fileprivate func setupLayout() {
+        secondRowStackView.removeAllArrangedViews()
+        thirdRowStackView.removeAllArrangedViews()
+        contentStackView.removeAllArrangedViews()
+
+        let contentSizeCategory = UIApplication.shared.preferredContentSizeCategory
+        if contentSizeCategory.isAccessibilityCategory {
+            setupLayoutForAccessibilityView()
+        } else {
+            setupLayoutForDefaultView()
+        }
+    }
+
+    private func setupLayoutForDefaultView() {
+        // first line
+        contentStackView.addArrangedSubview(titleLabel)
+
+        // second line
+        productLinkButton.setContentHuggingPriority(.required, for: .vertical)
+        productLinkButton.setContentCompressionResistancePriority(.required, for: .vertical)
+        productLinkButton.setContentHuggingPriority(.required, for: .horizontal)
+        productLinkButton.setContentCompressionResistancePriority(.required, for: .horizontal)
+
+        secondRowStackView.addArrangedSubview(productImageView)
+        secondRowStackView.addArrangedSubview(productLinkButton)
+
+        secondRowStackView.addArrangedSubview(gradeReliabilityScoreView)
+        secondRowStackView.distribution = .fillProportionally
+        secondRowStackView.alignment = .top
+
+        // third line
+        thirdRowStackView.addArrangedSubview(priceLabel)
+        thirdRowStackView.addArrangedSubview(spacer)
+        thirdRowStackView.addArrangedSubview(starRatingView)
+        thirdRowStackView.distribution = .fill
+        thirdRowStackView.alignment = .leading
+
+        priceLabel.heightAnchor.constraint(equalTo: starRatingView.heightAnchor, multiplier: 1.0).isActive = true
+        contentStackView.addArrangedSubview(secondRowStackView)
+        contentStackView.addArrangedSubview(thirdRowStackView)
+        contentStackView.distribution = .fillProportionally
+    }
+
+    private func setupLayoutForAccessibilityView() {
+        contentStackView.distribution = .fill
+        // first line
+        contentStackView.addArrangedSubview(titleLabel)
+
+        // second line
+        let spacerTrailing = UIView()
+        let secondLineStackView = UIStackView(arrangedSubviews: [productImageView, spacerTrailing])
+        secondLineStackView.axis = .horizontal
+        secondLineStackView.distribution = .fill
+        contentStackView.addArrangedSubview(secondLineStackView)
+
+        // third line
+        let spacerBottom = UIView()
+        let gradeStackView = UIStackView(arrangedSubviews: [starRatingView, gradeReliabilityScoreView, spacerBottom])
+        gradeStackView.axis = .horizontal
+        gradeStackView.spacing = UX.hStackSpacing
+        gradeStackView.distribution = .fill
+        gradeReliabilityScoreView.heightAnchor.constraint(equalTo: starRatingView.heightAnchor, multiplier: 1.0).isActive = true
```

Comment:
Same here: This view contains a label and labels should not have height constraints as this could lead to text being cut off when using large dynamic type.